src/cls/rgw/cls_rgw.cc

```diff
@@ -2019,7 +2016,7 @@ static int rgw_bucket_unlink_instance(cls_method_context_t hctx, bufferlist *in,
   ret = log_index_operation(hctx, op.key, CLS_RGW_OP_UNLINK_INSTANCE, op.op_tag,
                             mtime, ver,
                             CLS_RGW_STATE_COMPLETE, header.ver, header.max_marker,
-                            op.bilog_flags | RGW_BILOG_FLAG_VERSIONED_OP, NULL, NULL, &op.zones_trace);
+                            op.bilog_flags, NULL, NULL, &op.zones_trace);
```

Comment:
i know this looks safe because `RGWRados::unlink_obj_instance()` is adding this RGW_BILOG_FLAG_VERSIONED_OP flag now, but users might upgrade their osds before they upgrade their rgws. so if they're still using an rgw that doesn't contain that `RGWRados::unlink_obj_instance()` change, we'll omit this RGW_BILOG_FLAG_VERSIONED_OP flag

i think it's better to leave this line as it was, just to be on the safe side

Reply:
ah yes didn't think about that. thanks